letsencrypt/client/apache/configurator.py

```diff
@@ -523,6 +519,7 @@ def make_vhost_ssl(self, nonssl_vhost):
 
         return ssl_vhost
 
+    # pylint: disable=no-method-argument,no-self-use,unused-argument
```

Comment:
what's going on here? also, this probably removes pylint checking not only for the following method, but for the entire code block indented at this level...